app/views/trips/_current_trips.html.erb

```diff
@@ -11,10 +11,10 @@
       <div class="col-lg-4">
         <h1><%= trip.name %></h1>
         <p>
-          From: <%= trip.start_date if trip.start_date %>
+          From: <%= trip.start_date.strftime("%d/%m/%Y") if trip.start_date %>
```

Comment:
we can delete if statement in view cause we required a start and end date

Reply:
Totally right